components/log-viewer-webui/client/src/pages/IngestPage/sqlConfig.ts

```diff
@@ -0,0 +1,71 @@
+import axios from "axios";
+
+
+/**
+ * Sql table names.
+ */
+const SQL_CONFIG = Object.freeze({
+    SqlDbClpArchivesTableName: "clp_archives",
+    SqlDbClpFilesTableName: "clp_files",
+    SqlDbCompressionJobsTableName: "compression_jobs",
+});
+
+/**
+ * Enum of the column names for the `clp_archives` table.
+ *
+ * @enum {string}
+ */
+const CLP_ARCHIVES_TABLE_COLUMN_NAMES = Object.freeze({
+    BEGIN_TIMESTAMP: "begin_timestamp",
+    END_TIMESTAMP: "end_timestamp",
+    UNCOMPRESSED_SIZE: "uncompressed_size",
+    SIZE: "size",
+});
+
+/**
+ * Enum of the column names for the `clp_files` table.
+ *
+ * @enum {string}
+ */
+const CLP_FILES_TABLE_COLUMN_NAMES = Object.freeze({
+    ORIG_FILE_ID: "orig_file_id",
+    NUM_MESSAGES: "num_messages",
+});
+
+/**
+ * Enum of the column names for the `compression_jobs` table.
+ *
+ * @enum {string}
+ */
+const COMPRESSION_JOBS_TABLE_COLUMN_NAMES = Object.freeze({
+    CLP_BINARY_VERSION: "clp_binary_version",
+    CLP_CONFIG: "clp_config",
+    COMPRESSED_SIZE: "compressed_size",
+    CREATION_TIME: "creation_time",
+    DURATION: "duration",
+    ID: "id",
+    NUM_TASKS: "num_tasks",
+    NUM_TASKS_COMPLETED: "num_tasks_completed",
+    ORIGINAL_SIZE: "original_size",
+    START_TIME: "start_time",
+    STATUS: "status",
+    STATUS_MSG: "status_msg",
+    UNCOMPRESSED_SIZE: "uncompressed_size",
+    UPDATE_TIME: "update_time",
+});
```

Comment:
let's replace it with a TS enum

```suggestion
/**
 * Column names for the `compression_jobs` table.
 */
enum COMPRESSION_JOBS_TABLE_COLUMN_NAMES {
    CLP_BINARY_VERSION = "clp_binary_version",
    CLP_CONFIG = "clp_config",
    COMPRESSED_SIZE = "compressed_size",
    CREATION_TIME = "creation_time",
    DURATION = "duration",
    ID = "id",
    NUM_TASKS = "num_tasks",
    NUM_TASKS_COMPLETED = "num_tasks_completed",
    ORIGINAL_SIZE = "original_size",
    START_TIME = "start_time",
    STATUS = "status",
    STATUS_MSG = "status_msg",
    UNCOMPRESSED_SIZE = "uncompressed_size",
    UPDATE_TIME = "update_time",
}
```